src/client-entry.js

```diff
@@ -44,6 +45,27 @@ const {
 	kvAuth0,
 });
 
+// Show a tip message when there is an unhandled auth0 error
+kvAuth0.onError(({ eventId, user }) => {
+	let message = 'We\'re sorry, something went wrong.';
+	if (user) {
+		message = `${message} Please log out and try again.`;
+	} else {
+		message = `${message} Please clear your cookies and try again.`;
+	}
+	if (eventId) {
```

Comment:
Is the eventId only present when in "special" error scenarios?

Reply:
The event id should be present after any message is sent to sentry. I think the only real case it will not be present is on the vm where sentry is disabled.